src/data/feature.puml

```diff
@@ -0,0 +1,33 @@
+@startuml
+skinparam activity {
+BackgroundColor #76CDD8
+ArrowColor black
+ArrowFontSize 13
+ArrowFontStyle bold
+ArrowFontName Segoe UI
+BorderColor black
+FontName Segoe UI
+FontSize 15
+}
+(*) --> "Read ?"
```

Comment:
The "Feature" process just receives a `Feature` and processes it, there is no reading in it